### DISCUSSION TITLE
Deselecting row in <Table>

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#2821] [Q&A]
I am using onRowsDeselected property in Table component and it seems the this callback is only called when a selected row is deselected using Ctrl+click.  If a row is deselected when another row is selected, the callback function is not called.  Is this the expected behavior.  I am using SelectionMode.Extended.

Can I find out which rows of the table are currently selected?

Thanks,

Ovais

**Comment by NancyMcCallB** [comment]
Hi, Ovais,

We'll be deprecating the Table component in ui-components in the next release in favor of the Table component in [iTwinUI-react](https://github.com/iTwin/iTwinUI-react), because the ui-components Table component is based on a component that is not well-maintained any more. To future-proof your code, you'll want to look at the iTwinUI-react component.

Thanks,
Nancy McCall

**Reply by JohnMajerle** [reply]
Hey Nancy,
I'm assuming 'next release' means the 3.0 release.  Will there be a transition guide published to help us all port from the old API to the new?
Thanks,
John M.

**Reply by pmconne** [reply]
Yes.
